src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const list = tree.querySelectorAll('li');
+
+list.forEach((item) => {
+  const span = document.createElement('span');
+
+  item.prepend(span);
+  span.append(span.nextSibling);
+});
+
+tree.addEventListener('click', (e) => {
+  const item = e.target.nextSibling;
```

Comment:
You should check if the target is a 'span' element to ensure clicking outside the title text does nothing. Add 'if (e.target.tagName !== 'SPAN') return;' at the beginning of the event listener callback.